### PR TITLE
feat: write ordered maps by default

### PR DESCRIPTION
**Pull request by dwelch-spike**
TOOLS-2322
Write ordered maps by default.
Add the -um | --unorderedMaps flag to force maps to be written as unordered.
Upgrade Java client to 6.1.6
Fix broken geojson in example

### CODE REVIEW

File: doc/releasenotes.md [hunk 1]
```diff
@@ -59,4 +59,11 @@
...
 ### Fix/Changes:
--   Update log4j to version 2.17.1.
\ No newline at end of file
+-   Update log4j to version 2.17.1.
+
+## Aerospike Loader [3.0.0] Release Date [27 Jan 2023]
+### Breaking Changes:
+-   Upgrade Aerosppike Java client to 6.1.6 which is only compatible with Aerospike server 4.9 or newer.
+### New features:
+-   Write key ordered maps by default.
```
Comment (jdogmcsteezy) [inline doc/releasenotes.md:68]:
```suggestion
-   Upgrade Aerosppike Java client to 6.1.6 which is only compatible with Aerospike server 4.9 or newer.
-   Write key ordered maps by default.
### New features:
```

File: doc/releasenotes.md [hunk 2]
```diff
@@ -59,4 +59,11 @@
...
 ### Fix/Changes:
--   Update log4j to version 2.17.1.
\ No newline at end of file
+-   Update log4j to version 2.17.1.
+
+## Aerospike Loader [3.0.0] Release Date [27 Jan 2023]
+### Breaking Changes:
+-   Upgrade Aerospike Java client to 6.1.6 which is only compatible with Aerospike server 4.9 or newer.
+### New features:
+-   Write key ordered maps by default.
```
Comment (jdogmcsteezy) [inline doc/releasenotes.md]:
```suggestion
-   Upgrade Aerospike Java client to 6.1.6 which is only compatible with Aerospike server 4.9 or newer.
-   Write key ordered maps by default.
### New features:
```

File: doc/releasenotes.md (hunk 1 above)
Comment (dwelch-spike) [reply doc/releasenotes.md]: Fixed a typo that caused this to go out of date (unmergable) but did make the change.

**Review by jdogmcsteezy — APPROVED** [review]
LGTM

**Pull request merged**